pyroma/ratings.py

```diff
@@ -225,14 +226,15 @@ def message(self):
         return "Some of your classifiers are not standard classifiers:\n" + err
 
 
-class PythonVersion(BaseTest):
+class PythonClassifierVersion(BaseTest):
+
     def test(self, data):
         self._major_version_specified = False
 
-        classifiers = data.get("classifiers", [])
+        classifiers = data.get('classifiers', [])
```

Comment:
So this is just changing which quotes to use, which seems very spurious. Anyway, I realized through this that I hadn't enforced `black` so I did that.

Please rebase this branch from master, and black should refuse this change. Sorry I hadn't enforced it earlier.

Reply:
done.